Documentation/Storage-Configuration/NFS/nfs.md

```diff
@@ -0,0 +1,175 @@
+---
+title: NFS Storage Overview
+---
+
+NFS storage can be mounted with read/write permission from multiple pods. NFS storage may be
+especially useful for leveraging an existing Rook cluster to provide NFS storage for legacy
+applications that assume an NFS client connection. Such applications may not have been migrated to
+Kubernetes or might not yet support PVCs. Rook NFS storage can provide access to the same network
+filesystem storage from within the Kubernetes cluster via PVC while simultaneously providing access
+via direct client connection from within or outside of the Kubernetes cluster.
+
+!!! warning
+    Simultaneous access to NFS storage from Pods and from from external clients complicates NFS user
```

Comment:
What is the complication? Are we saying it's not recommended, or is there a link to deal with the complication?

Reply:
The complication is that it might make CSI unable to mount exports for pods. I don't know a workaround currently. Seems good to clarify this point.

Reply:
Oh, wait. I misread this note. I thought it was one above. 

It's kind of hard to explain... I think the simplest gist is that k8s containers aren't usually given the same user as one would use for a server using an NFS mount, and the tools used (e.g., SSSD) to automatically get user data from an LDAP server are cumbersome to configure for pods. I guess it comes down to the idea that ID mapping doesn't fit well into the pod/container space, but it is required for most pre-existing NFS shares.

Reply:
How about if we just recommend they configure single user access since multiple user management is more complex? Some sort of wording like that would help understand that the complexity is around multiple user management.